docs/spfx/toolchain/implement-ci-cd-with-azure-pipelines.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Implement Continuous Integration and Continuous deployment using Azure Pipelines (preview)
+description: Streamlining the build and deployment process by automating manual steps.
+ms.date: 12/03/2019
+ms.prod: sharepoint
+localization_priority: Priority
+---
+# Implement Continuous Integration and Continuous deployment using Azure Pipelines
+Azure Pipelines is the newer version of the Azure DevOps build and release features.  
+
+This article explains the steps involved in setting up your Azure Pipelines environment with Continuous Integration and Continuous Deployment to automate your SharePoint Framework builds, unit tests, and deployment.  
+
+> [!NOTE] 
+> Azure Multi-stage Pipelines is still in preview and you need to make sure you enable this preview feature for your Azure DevOps organization under _Preview Features_
+
+## Chosing between Azure Multi-stage Pipelines (preview) and Azure DevOps builds and releases
+There are currently two approaches available to implement Continuous Integration, Testing and Deployement in Azure DevOps.  
```

Comment:
CI/CD should be lower case, not caps. lcase is how you have it in other places in this PR

Also... isn't "testing" part of integration? Never seen it sep. out as it's own thing... let's not introduce it as something new here.

Reply:
fixed